otherlibs/stdune/src/string.ml

```diff
@@ -221,12 +231,6 @@ let enumerate_one_of = function
   | s -> "One of " ^ enumerate_or s
 ;;
 
-let concat ~sep = function
-  | [] -> ""
-  | [ x ] -> x
-  | xs -> concat ~sep xs
-;;
```

Comment:
This function exists in the stdlib in all OCaml versions we support with the same semantics  and only works because it accidentally calls `Stdlib.StringLabels.concat` (missing the `rec` flag); otherwise it never uses the `sep` argument.